root/static/scripts/edit/URLCleanup.js

```diff
@@ -2830,7 +2830,7 @@ const CLEANUPS = {
         case LINK_TYPES.streamingfree.release:
           return {
             error: l(
-              'Only video and playlist links are allowed on releases.',
+              'Only video and playlist links are valid on releases.',
```

Comment:
This has more of a legitimate reason to be called valid, although I still think "allowed" leaves a clearer impression that we just don't want that link.

Reply:
Just like above, my concern is probably more about the missing why.

Reply:
Hmm. We have never specified the why before, but I guess this could be something like               'Only video and playlist links are valid on releases. Channel / user pages should be added to the artist, label, etc. as appropriate.' ? Of course, maybe they've pasted something that isn't a channel or user page either.

Reply:
Since I don’t have a better proposition, I just dropped that change.